skimage/segmentation/active_contour_model.py

```diff
@@ -7,11 +6,11 @@
 
 def active_contour(image, snake, alpha=0.01, beta=0.1,
                    w_line=0, w_edge=1, gamma=0.01,
-                   bc=None, max_px_move=1.0,
+                   max_px_move=1.0,
                    max_iterations=2500, convergence=0.1,
                    *,
                    boundary_condition='periodic',
-                   coordinates=None):
+                   coordinates='rc'):
```

Comment:
If I understood correctly the original motivation, the idea was to remove this argument once the deprecation is over - https://github.com/scikit-image/scikit-image/pull/4035 .

Reply:
Thank you!

Reply:
My concern is: if I remove the parameter, it will break the user code because the message said to put the option ='rc' to suppress the warning.
Ideally, we need another deprecation round to raise a warning and say, to suppress it, remove this kw arg.

@jni any comment on your original thought?

Reply:
So we would need to add another deprecation message here?

Reply:
the gallery example `edges/plot_active_contour.py` uses `coordinates='rc'` and should be modified too.

Reply:
yes

Reply:
@emmanuelle @soupault as per my comment below, I think this argument should stay but it should do nothing, other than maybe raise a ValueError when set to anything other than 'rc'. Indeed, it *must* raise a ValueError to remind anyone using `coordinates='xy'` that their code is no longer valid.